OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -128,19 +137,19 @@ public void Tick(World world)
 				target = blockedPos;
 			}
 
-			if (!doneDamage)
+			if (ticks < info.DamageDuration && --interval <= 0)
```

Comment:
This doesn't seem to match the `[Desc()]` above: if DamageInterval is 5 and DamageDuration is 2 then I would expect ticks 0, 1 to deal damage, then ticks 5, 6, and so on.  This looks like it will disable all damage after the first `DamageDuration` ticks.

Reply:
It works like I intended it to, seems the desc/property name aren't accurate enough then. Not sure what else to call it though, `DamageTimeWindow`?

Reply:
I settled for making the description more clear for now, since the behavior of DamageDuration matches the beam Duration (i.e. it stands for total time-frame, not a time-frame every n ticks).